### PR TITLE
chore(goss): migrating shell sanity tests to goss - phase 1

### PR DESCRIPTION
**Pull request by smerle33**
as per https://github.com/jenkins-infra/helpdesk/issues/3763

need PR https://github.com/jenkins-infra/packer-images/pull/896 after to update the updatecli manifests to the new goss file as a target

### CODE REVIEW

File: updatecli/updatecli.d/azure-cli.yml [hunk 1]
```diff
@@ -31,12 +31,19 @@ sources:
 
 targets:
   updateVersion:
-    name: "Update the azure-cli version in the provision-env.yml file"
+    name: "Update the `azure-cli`` version in the provision-env.yml file"
```
Comment (dduportal) [inline updatecli/updatecli.d/azure-cli.yml:34]:
```suggestion
    name: "Update the `azure-cli` version in the provision-env.yml file"
```

File: updatecli/updatecli.d/awscli.yml [hunk 2]
```diff
@@ -41,7 +41,14 @@ targets:
...
     spec:
       file: "provisioning/tools-versions.yml"
-      key: "awscli_version"
+      key: "$.awscli_version"
+    scmid: default
+  updateVersionInGoss:
+    name: Update the `aws` CLI  version in the goss test
+    kind: yaml
+    spec:
+      file: "goss/goss.yaml"
```
Comment (smerle33) [inline updatecli/updatecli.d/awscli.yml:50]: need to rename in all files !

Comment (smerle33) [reply updatecli/updatecli.d/awscli.yml:50]: in PR #896

**Review by dduportal — APPROVED** [review]
👏

**Pull request merged**